frontend/src/app/modules/attachments/attachments.component.ts

```diff
@@ -36,8 +36,10 @@ import {States} from 'core-components/states.service';
 import {componentDestroyed} from 'ng2-rx-componentdestroyed';
 import {filter, takeUntil} from 'rxjs/operators';
 
+export const attachmentsSelector = 'attachments';
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/be03a9ed2bd960efe9bf08e68a4088034cb73cd9/frontend/src/app/modules/attachments/attachments.component.ts#L40"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>